packages/blade/src/tokens/theme/theme.d.ts

```diff
@@ -29,13 +29,35 @@ export type Shadows = {
   };
 };
 
-export type Feedback = 'information' | 'negative' | 'notice' | 'positive';
+export type Feedback = 'information' | 'negative' | 'neutral' | 'notice' | 'positive';
```

Comment:
Seems like some merge issue.
Can you sync with latest master?

Reply:
no merge issue. I've added new value and disabled the sorting of keys eslint rule hence you see this weird addition